### PR TITLE
Helpful hours screen 2 Updated code and re-usable components

### PR DESCRIPTION
**Pull request by ttran0012**
Hi, This is the updated helpful hour's screen with re-usable code. Please take a look and make any changes neccessary. At this point you can make your own. :) that's fine with me

### CODE REVIEW

File: screens/HelpfulHours.jsx [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
+
 export default HelpfulHours;
 
 const styles = StyleSheet.create({
     center: {
         flex: 1,
+        paddingHorizontal: 18,
         justifyContent: "center",
-        alignItems: "center",
+        //alignItems: "center",
```
Comment (Falzaro) [inline screens/HelpfulHours.jsx:63]: Next time, please remove the comments that we don't need in your pull requests.

**Comment by Falzaro** [comment]
These changes look much better and are more reusable. Since the backend will need to change the data structures of Helpful hours, this pull request will get affected. We'll need to resolve the Firestore issue first before I can merge this PR.

**Comment by ttran0012** [comment]
Can you give me the algorithm to extract the data structure? If you can figure it out w/in a matter of seconds. Then that would save me on my time. I don't want to sit for 8 hours and get nothing done.

**Comment by ttran0012** [comment]
Resolved Merge Conflict

**Pull request merged**